packages/pong-engine/src/queue.dto.ts

```diff
@@ -6,6 +6,12 @@ export type GameChallengeDto = {
   };
 };
 
+export type GameUserChallengeDto = {
```

Comment:
I agree. This needs a different package. But I'm having a nightmarish difficult time building and compiling the packages every time (nothing consistently works. No `make re`, no pruning the volumes, no deleting the `packages/*/lib` folders) I have to keep compiling and recompiling the project over and over again until eventually by random chance it finally compiles the package I need and both webapp and transcendence-app can find the symbols. This one is already huuuge withouot this. Do you mind if I leave this for a later PR? I will open a ticket on the topic.

Reply:
Opened ticket [here](https://github.com/42AGV/ft_transcendence/issues/476).

Reply:
No worries, this can be improved later. Thanks!

We may also review the rebuilding of the package issues you mention. I think the only solution could be having some kind of hot-reload